graph/src/components/link_resolver.rs

```diff
@@ -1,9 +1,12 @@
 use data::subgraph::Link;
 use failure;
 use ipfs_api;
+use std::env;
+use std::str::FromStr;
+use std::time::Duration;
 use tokio::prelude::*;
 
-use std::time::Duration;
+const MAX_IPFS_FILE_BYTES_ENV_VAR: &str = "GRAPH_MAX_IPFS_FILE_BYTES";
```

Comment:
How about making this a lazy_static with the parsed value? That way a parsing error will panic on start up, instead of going unnoticed

Reply:
Ah, might need to make it a parameter on IpfsClient then for testing

Reply:
lazy_static is lazy so it wouldn't help with failing earlier. Though you make a good point that an env var failing to parse should be a fatal error, I've changed that to panic.